src/Microsoft.Health.SqlServer.Api/Controllers/SchemaController.cs

```diff
@@ -45,10 +48,10 @@ public ActionResult AvailableVersions()
 
             var availableSchemas = new List<object>();
             var currentVersion = _schemaInformation.Current ?? 0;
-            foreach (var version in Enum.GetValues(typeof(SchemaVersion)).Cast<SchemaVersion>().Where(sv => sv >= currentVersion))
+            foreach (var version in Enum.GetValues(typeof(TSchemaVersionEnum)).Cast<int>().Where(sv => sv >= currentVersion))
```

Comment:
A little bit of thinking out loud here, but could this be driven from ISchemaInformation now? i.e. `for(int i = schemaInformation.Current; i <= schemaInformation.Max...)`?